lib/fs/basicfs_watch_test.go

```diff
@@ -208,6 +208,54 @@ func TestWatchErrorLinuxInterpretation(t *testing.T) {
 	}
 }
 
+func TestWatchSymlinkedRoot(t *testing.T) {
+	if runtime.GOOS == "windows" {
+		t.Skip("Involves symlinks")
+	}
```

Comment:
There is a better way to check this somewhere.

Reply:
i know bit it's in osutil and that imports fs so it's cyclic. I didn't consider it worth working around that.